src/task.rs

```diff
@@ -1098,6 +1098,12 @@ pub struct TaskMetrics {
     /// ```
     pub total_fast_poll_count: u64,
 
+    /// The total number of times that a task was executed swiftly.
+    ///
+    /// Here, 'swiftly' is defined as executing in strictly less time than
+    /// [`long_delay_threshold`][TaskMonitor::long_delay_threshold].
```

Comment:
You've used "swiftly" here and "quickly" below.

It might be a good idea to follow the form used for the `long` metrics. Which would make this:

```suggestion
    /// The total count of tasks with short scheduling delays.
    ///
    /// This is defined as tasks taking strictly less than
    /// [`long_delay_threshold`][TaskMonitor::long_delay_threshold] to be executed after being
    /// scheduled.
```